packages/desktop-client/src/components/ThemeSelector.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { useActions } from '../hooks/useActions';
+import MoonStars from '../icons/v2/MoonStars';
+import Sun from '../icons/v2/Sun';
+import { useResponsive } from '../ResponsiveProvider';
+import { useTheme } from '../style';
+
+import { Button, View } from './common';
+
+export function ThemeSelector() {
+  let theme = useTheme();
+  let { saveGlobalPrefs } = useActions();
+
+  let { isNarrowWidth } = useResponsive();
+
+  // Don't display on mobile
+  return isNarrowWidth ? null : (
+    <View>
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏do we need the wrapping `View`?